src/com/jetbrains/Duke.java

```diff
@@ -2,7 +2,6 @@
 
 import java.util.Scanner;
 import java.util.ArrayList;
-import java.lang.Throwable;
 
 public class Duke {
```

Comment:
Maybe some java docs will be useful to make the user understand the code better!